src/element/Hyperlink.tsx

```diff
@@ -203,11 +202,11 @@ const getCoordsForPopover = (
 
 export const getAbsoluteLink = (link?: string) => {
   if (link) {
-    const match = VALID_PREFIXES.find((prefix) => link!.startsWith(prefix));
+    const protocolIndex = link.indexOf("://");
 
-    // prefix with https if no match
-    if (!match) {
-      link = `${VALID_PREFIXES[0]}${link}`;
+    // prefix with https if no protocol
+    if (protocolIndex === -1) {
```

Comment:
I don't want to over complicate the situation... but it would be great if you would allow `[[wiki links]]`. 
maybe use the following logic: 
```typescript
if (protocolIndex === -1 && link[0]!== "[") {
  link = `https://${link}`;
}
```

Reply:
Not sure how it'll work for you in Obsidian, but I think this is acceptable. But, we may use the same or similar syntax for on-canvas elements (ids), something like `[[element <id>]]`.

I'd change the code to:

```
if (!link.includes("://") && !link.startsWith("[")) {
```